src/components/Registration/BusinessAddresses.vue

```diff
@@ -404,9 +396,7 @@ label {
 // }
 
 // italicize delivery instructions
-::v-deep .base-address {
-  .address-block__info-row:last-of-type {
-    font-style: italic;
```

Comment:
Oops, this italicized the country if there was no delivery instructions. But only for registrations, which were not deployed yet 😁